basis/formatting/formatting.factor

```diff
@@ -113,9 +113,9 @@ fmt-C     = "C"                  => [[ [ 1string >upper ] ]]
 fmt-s     = "s"                  => [[ [ present ] ]]
 fmt-S     = "S"                  => [[ [ present >upper ] ]]
 fmt-u     = "u"                  => [[ [ unparse ] ]]
-fmt-d     = "d"                  => [[ [ >integer number>string ] ]]
-fmt-o     = "o"                  => [[ [ >integer >oct ] ]]
-fmt-b     = "b"                  => [[ [ >integer >bin ] ]]
+fmt-d     = "d"                  => [[ [ number>string ] ]]
+fmt-o     = "o"                  => [[ [ >oct ] ]]
+fmt-b     = "b"                  => [[ [ >bin ] ]]
 fmt-e     = digits "e"           => [[ first '[ _ format-scientific ] ]]
```

Comment:
I think it should mean "print this as an integer"... essentially the same as ``%.0f``.  Otherwise, it should throw an error for non-integer inputs (which would be weird, too).  What it shouldn't do is print ``1.23456789``...